src/main/webapp/vendor/startbootstrap/index.html

```diff
@@ -18,6 +18,11 @@
   <!-- Custom styles for this template-->
   <link href="https://cdn.jsdelivr.net/npm/startbootstrap-sb-admin-2@4.1.1/css/sb-admin-2.min.css" rel="stylesheet">
 
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
```

Comment:
Remove this stylesheet and the all of the issues with the rest of the UI go away